shared/src/iosMain/kotlin/com/kcc/kmmhackathon/shared/utility/DateFormattingUtil.kt

```diff
@@ -0,0 +1,16 @@
+package com.kcc.kmmhackathon.shared.utility
+
+import platform.Foundation.*
+
+actual class DateFormattingUtil actual constructor() {
+
+    actual fun convertTimeStamp(timeStamp: Long): String {
+        val date = NSDate.dateWithTimeIntervalSince1970(timeStamp.toDouble())
+        formatter.dateFormat = "E d MMM"
+        return formatter.stringFromDate(date)
+    }
+
+    actual companion object {
+        val formatter = NSDateFormatter()
```

Comment:
If I am understanding correctly, companion objects are singletons and thus the DateFormatter() will be reused here.
Let me know if I got that wrong 😬

Reply:
That's correct :) it's similar to doing `static let formatter = NSDateFormatter()` in a Swift type